lib/tasks/configuration/generate_for_schools.rake

```diff
@@ -0,0 +1,15 @@
+namespace :configuration do
+  desc 'Run alerts job'
+  task generate_for_schools: [:environment] do
+    puts Time.zone.now
+    School.all.each do |school|
+      if school.meters_with_readings.any?
```

Comment:
Can we let the configuration decide this for us? Feels like we're leaking logic out to the rake task. Will it always hold that we don't want to run this for schools without readings?

Reply:
Handle exception in the rake task if something goes wrong in the generation, so it doesn't just stop.